modules/applications/pages/uid-applications-index.adoc

```diff
@@ -0,0 +1,105 @@
+= Living Applications with Bonita UID
+:description: This page explains what a Living Application is, what it is made of, and how you can create your own applications.
+
```

Comment:
As you rename this page, the :page-aliases: are mandatory and refer to the oldest page name.

```suggestion
:page-aliases: applications:custom-applications-index.adoc
```